### DISCUSSION TITLE
Welcome to telegram-upload Discussions!

### DISCUSSION THREAD

**Discussion by Nekmo** [Nekmo/telegram-upload#80] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eN0Rm** [comment]
Can one upload to a group?

**Reply by Nekmo** [reply]
Yes! Use the group invite link:

https://docs.nekmo.org/telegram-upload/usage.html#cmdoption-telegram-upload-to
